docker-compose.yml

```diff
@@ -17,7 +17,7 @@ hydroshare:
     PGPASSWORD: postgres
   volumes:
     # hydroshare repository
-    - ".:/home/docker/hydroshare"
+    - "/home/hydro/hydroshare:/home/docker/hydroshare"
```

Comment:
This change is constantly breaking our development environments here and `"."` should work for everyone. Configuration should not depend on global setup of a development machine wherever it is avoidable.